<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ZTelosControlDialog</class>
    <widget class="QDialog" name="ZTelosControlDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>681</width>
                <height>550</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>681</width>
                <height>550</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Select zBWK to Spend</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_1" stretch="0">
            <property name="spacing">
                <number>0</number>
            </property>
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item>
                <widget class="QFrame" name="frame">
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_2">
                        <property name="leftMargin">
                            <number>0</number>
                        </property>
                        <property name="topMargin">
                            <number>12</number>
                        </property>
                        <property name="rightMargin">
                            <number>0</number>
                        </property>
                        <property name="bottomMargin">
                            <number>20</number>
                        </property>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout_12">
                                <property name="spacing">
                                    <number>0</number>
                                </property>
                                <property name="leftMargin">
                                    <number>20</number>
                                </property>
                                <property name="rightMargin">
                                    <number>30</number>
                                </property>
                                <item>
                                    <spacer name="horizontalSpacer_1">
                                        <property name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>40</width>
                                                <height>20</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                                <item>
                                    <widget class="QLabel" name="labelTitle">
                                        <property name="minimumSize">
                                            <size>
                                                <width>0</width>
                                                <height>40</height>
                                            </size>
                                        </property>
                                        <property name="maximumSize">
                                            <size>
                                                <width>16777215</width>
                                                <height>40</height>
                                            </size>
                                        </property>
                                        <property name="styleSheet">
                                            <string notr="true">padding-left:30px;</string>
                                        </property>
                                        <property name="text">
                                            <string>Coin Control</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                        <property name="margin">
                                            <number>7</number>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <spacer name="horizontalSpacer_2">
                                        <property name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>40</width>
                                                <height>20</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                                <item>
                                    <widget class="QPushButton" name="btnEsc">
                                        <property name="minimumSize">
                                            <size>
                                                <width>24</width>
                                                <height>24</height>
                                            </size>
                                        </property>
                                        <property name="maximumSize">
                                            <size>
                                                <width>24</width>
                                                <height>24</height>
                                            </size>
                                        </property>
                                        <property name="text">
                                            <string>PushButton</string>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
                                <property name="spacing">
                                    <number>-1</number>
                                </property>
                                <property name="leftMargin">
                                    <number>20</number>
                                </property>
                                <property name="rightMargin">
                                    <number>20</number>
                                </property>
                                <property name="bottomMargin">
                                    <number>10</number>
                                </property>
                                <item>
                                    <widget class="QWidget" name="layoutAmount" native="true">
                                        <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,1">
                                            <item>
                                                <widget class="QLabel" name="labelZTelos">
                                                    <property name="text">
                                                        <string>zTelos</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QLabel" name="labelZTelos_int">
                                                    <property name="text">
                                                        <string>0.00 </string>
                                                    </property>
                                                </widget>
                                            </item>
                                        </layout>
                                    </widget>
                                </item>
                                <item>
                                    <spacer name="horizontalSpacer_3">
                                        <property name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                        </property>
                                        <property name="sizeType">
                                            <enum>QSizePolicy::Fixed</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>40</width>
                                                <height>20</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                                <item>
                                    <widget class="QWidget" name="layoutQuantity" native="true">
                                        <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,1">
                                            <item>
                                                <widget class="QLabel" name="labelQuantity">
                                                    <property name="text">
                                                        <string>Quantity:</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QLabel" name="labelQuantity_int">
                                                    <property name="text">
                                                        <string>0</string>
                                                    </property>
                                                </widget>
                                            </item>
                                        </layout>
                                    </widget>
                                </item>
                                <item>
                                    <spacer name="horizontalSpacer_4">
                                        <property name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>40</width>
                                                <height>20</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout3" stretch="0,0">
                                <property name="spacing">
                                    <number>0</number>
                                </property>
                                <property name="leftMargin">
                                    <number>20</number>
                                </property>
                                <property name="rightMargin">
                                    <number>20</number>
                                </property>
                                <property name="bottomMargin">
                                    <number>10</number>
                                </property>
                                <item>
                                    <widget class="QPushButton" name="pushButtonAll">
                                        <property name="minimumSize">
                                            <size>
                                                <width>180</width>
                                                <height>40</height>
                                            </size>
                                        </property>
                                        <property name="text">
                                            <string>Select/Deselect All</string>
                                        </property>
                                        <property name="checkable">
                                            <bool>true</bool>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <spacer name="horizontalSpacer_5">
                                        <property name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>40</width>
                                                <height>20</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QVBoxLayout" name="verticalLayout_3">
                                <property name="leftMargin">
                                    <number>20</number>
                                </property>
                                <property name="rightMargin">
                                    <number>20</number>
                                </property>
                                <item>
                                    <widget class="QTreeWidget" name="treeWidget">
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                                                <horstretch>0</horstretch>
                                                <verstretch>0</verstretch>
                                            </sizepolicy>
                                        </property>
                                        <property name="minimumSize">
                                            <size>
                                                <width>0</width>
                                                <height>250</height>
                                            </size>
                                        </property>
                                        <property name="alternatingRowColors">
                                            <bool>true</bool>
                                        </property>
                                        <property name="sortingEnabled">
                                            <bool>true</bool>
                                        </property>
                                        <property name="columnCount">
                                            <number>6</number>
                                        </property>
                                        <attribute name="headerDefaultSectionSize">
                                            <number>100</number>
                                        </attribute>
                                        <column>
                                            <property name="text">
                                                <string notr="true">Select</string>
                                            </property>
                                        </column>
                                        <column>
                                            <property name="text">
                                                <string notr="true">Denomination</string>
                                            </property>
                                        </column>
                                        <column>
                                            <property name="text">
                                                <string notr="true">zBWK ID</string>
                                            </property>
                                        </column>
                                        <column>
                                            <property name="text">
                                                <string notr="true">zBWK Version</string>
                                            </property>
                                        </column>
                                        <column>
                                            <property name="text">
                                                <string notr="true">Confirmations</string>
                                            </property>
                                        </column>
                                        <column>
                                            <property name="text">
                                                <string>Is Spendable</string>
                                            </property>
                                        </column>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout_2">
                                <property name="spacing">
                                    <number>0</number>
                                </property>
                                <property name="leftMargin">
                                    <number>20</number>
                                </property>
                                <property name="rightMargin">
                                    <number>26</number>
                                </property>
                                <item>
                                    <spacer name="horizontalSpacer_6">
                                        <property name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>558</width>
                                                <height>20</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                                <item>
                                    <widget class="QDialogButtonBox" name="buttonBox">
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                                                <horstretch>0</horstretch>
                                                <verstretch>0</verstretch>
                                            </sizepolicy>
                                        </property>
                                        <property name="minimumSize">
                                            <size>
                                                <width>150</width>
                                                <height>35</height>
                                            </size>
                                        </property>
                                        <property name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                        </property>
                                        <property name="standardButtons">
                                            <set>QDialogButtonBox::Ok</set>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>accepted()</signal>
            <receiver>ZTelosControlDialog</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>248</x>
                    <y>254</y>
                </hint>
                <hint type="destinationlabel">
                    <x>157</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>buttonBox</sender>
            <signal>rejected()</signal>
            <receiver>ZTelosControlDialog</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>316</x>
                    <y>260</y>
                </hint>
                <hint type="destinationlabel">
                    <x>286</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
